gtsfm/utils/metrics.py

```diff
@@ -451,3 +453,70 @@ def compute_percentage_change(x: float, y: float) -> float:
         percentage change (may be positive or negative).
     """
     return (y - x) / (x + EPSILON) * 100
+
+
+def compute_psnr_after_downsampling(
```

Comment:
I am not sure this method belongs in metrics.py. Maybe its better in mvs_utils?

Reply:
I agree. I have put these two methods to mvs_utils.